### PR TITLE
[NPU] fix set_value, test=develop

### PR DESCRIPTION
**Pull request by qili93**
### PR types

Bug fixes

### PR changes

OPs

### Describe

Fix set_value OP for NPU.

### CODE REVIEW

File: paddle/fluid/operators/set_value_op_npu.cc [hunk 1]
```diff
@@ -1,319 +1,47 @@
...
-      slice_dims_with_none.push_back(slice_dims[i]);
-    }
-    // if the none_axes.size() > slice_dims.size(), append 1 after last dim
-    while (none_axes_cur < none_axes.size()) {
-      slice_dims_with_none.push_back(1);
-      none_axes_cur++;
-    }
-    slice_dims = slice_dims_with_none;
-  }
+using Tensor = Tensor;
```
Comment (zhhsplendid) [inline paddle/fluid/operators/set_value_op_npu.cc]: why `using Tensor = Tensor;` ?

Comment (qili93) [reply paddle/fluid/operators/set_value_op_npu.cc]: typo, fixed as "using Tensor = framework::Tensor;"

Comment (qili93) [reply paddle/fluid/operators/set_value_op_npu.cc]: removed as duplicated in set_value_op.h, and removed

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**